qgis-projects/README.md

```diff
@@ -0,0 +1,46 @@
+# QGIS Projects
+
+- dominode-topomaps.qgs
+  - Custodian: charles@kartoza.com
+  - Description: Primary map project for toposheet production
+
+## Project data
+
+Adding data should be done directly by primary project custodians wherever possible. Adding of features or layers to existing data sources (remote/ gpkg etc) should be distributed in the same manner as the original data source (republished) or have their source details modified in the following section.
+
+---
+
+- File: ./data/data.gpkg
+- Custodian: Default
+- Description: Local copy of utilised data copied from remote staging database as well as additional features sourced from public internet sources.
+- Source: [Download from google drive](https://drive.google.com/file/d/1B3o7bPx1iQH5qMXUGWQO-BwLkeJYlYP7/view?usp=sharing).
+
+> Note that the *data* directory is intended to be a symbolic link to a local data directory separate from the project repository.
```

Comment:
Format this as normal text (instead of a quote) in order to be more harmonized with the rest of the document